Example/ViewController.swift

```diff
@@ -20,6 +20,33 @@ class ViewController: UIViewController {
         player.attachTo(playerContainer, controller: self)
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        NotificationCenter.default.addObserver(self, selector: #selector(rotated), name: .UIDeviceOrientationDidChange, object: nil)
+    }
+
+    func rotated() {
+        if (player?.isFullscreen)! && (UIDevice.current.orientation == .faceUp || UIDevice.current.orientation == .faceDown) {
```

Comment:
Could we change it to a guard let in order to remove the bang and so avoiding the possibility of some crash at runtime?

Reply:
This code is in our sample app, not in the Clappr itself. Should we really change it?

Reply:
@leandroalonso  I agree with you, but i change the implementation to be more clear about what the method does.